openfe/setup/campaigners/easy_campaigner.py

```diff
@@ -0,0 +1,180 @@
+
+class rhfe_campaigner(relative_campaigner):
+    def __init__(self, mapper: AtomMapper = LomapAtomMapper,
+                 mapping_scorers: Iterable[Callable] = [default_lomap_score],
+                 networker: Callable = minimal_spanning_graph):
+        super().__init__(mapper=mapper, mapping_scorers=mapping_scorers,
+                         networker=networker, protocol=RelativeLigandTransform)
+
+    def __call__(self,  ligands:Iterable[SmallMoleculeComponent], solvent: SolventComponent = None,)->AlchemicalNetwork:
+        # components might be given differently!
+
+        # throw into Networker
+        # TODO: decompose networker here into the nice helpers above
+        self.ligand_network = self.networker(ligands=ligands,
+                                             mappers=[self.mapper],
+                                             mapping_scorers=self.mapping_scorers)
+
+        # Prepare system generation
+        self.chemical_system_generator = self.generator_type(solvent=solvent,
+                                                             do_vacuum=True)
+
+        # build transformations
+        self.transformer = rfe_transformer(protocol=self.protocol,
+                                           system_generator=self.chemical_system_generator)
+        self.alchemical_network = self.transformer(alchemical_network_edges=self.ligand_network.edges)
+
+        return self.alchemical_network
+
+class rbfe_campaigner(relative_campaigner):
+
+    def __init__(self, mapper: AtomMapper = LomapAtomMapper,
+                 mapping_scorers: Iterable[Callable] = [default_lomap_score],
+                 networker: Callable = minimal_spanning_graph):
+        super().__init__(mapper=mapper, mapping_scorers=mapping_scorers,
+                         networker=networker, protocol=RelativeLigandTransform)
+
+    def __call__(self, ligands:Iterable[SmallMoleculeComponent], solvent: SolventComponent = None, receptor: ProteinComponent = None,):
+        # components might be given differently!
+
+        # throw into Networker
+        # TODO: decompose networker here into the nice helpers above
+        self.ligand_network = self.networker(ligands=ligands,
+                                             mappers=[self.mapper],
+                                             mapping_scorers=self.mapping_scorers)
+
+        # Prepare system generation
+        self.chemical_system_generator = self.generator_type(solvent=solvent,
+                                                             protein=receptor)
+
+        # build transformations
+        self.transformer = rfe_transformer(protocol=self.protocol,
+                                           system_generator=self.chemical_system_generator)
+        self.alchemical_network = self.transformer(alchemical_network_edges=self.ligand_network.edges)
+
+        return self.alchemical_network
+
+
+"""
+    easy absolute campaigner
+"""
+class absolute_campaigner(_abstract_campaigner):
```

Comment:
So that we can actually merge this PR this month, can we ask that we don't have the absolute bits yet? The way the protocol is going to behave will be different, and I'd prefer not have to either a) have to shoehorn the protocol to fit this, or b) have to see a bunch of work go here and then have to kill it because it doesn't fit the way the protocol works.

Reply:
sorry , that was discussed earlier in the meeting! this was not plan to stay here rather to show case the idea;)